### PR TITLE
improve accuracy and readability of outbound api docs

### PR DESCRIPTION
**Pull request by jschatzman-lp**

### CODE REVIEW

**Review by lp-kmehta — COMMENTED** [review]
Overall looks good! I have added a few comments. But we are making more changes to the reporting API we will need to consistently keep this updated.

**Comment by itay1313** [comment]
Let me know if you want me to merge it

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ redirect_from:
 
 ### Introduction
 
-Outbound Reporting product provides a complete message journey of the conversation from start to finish. The Outbound Reporting API provides a full funnel metrics number of messages/deflections sent, how many actually got delivered by the messaging channel, how many were read and responded back by the consumers. The goal of this feature is to stitch pre and post conversation events of every message/deflections and provide 360° analytical data to API clients.
+The Outbound Reporting API provides a complete message journey of the conversation from start to finish. It provides a full end to end summary of messages/deflections sent, including how many actually got delivered by the messaging channel, how many were read and responded back by the consumers. This service stitches pre and post conversation events of every message/deflections and provide 360° analytical data to API clients.
```
[the remark this answers is not in the document]
Comment (jschatzman-lp) [reply pages/Documents/Reporting/OutboundReportingAPI/overview.md:16]: Done

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md [hunk 2]
```diff
@@ -27,26 +27,25 @@ All Brands who use Proactive Messaging version 2.0 and Connect To Messaging vers
...
 **Feature Details**
 
 Outbound Reporting API provides the following data fields. This table explains the definition of each field.
 
 | # | Data Field | Definition |
-| 1 | Attempted | Total Outbound Messages/Total IVR Deflections sent to the Proactive Messaging/Connect To Messaging system |
-| 2 | Eligible | From total attempted messages how many consumers are eligible to receive  messages. |
-| 3 | In-Eligible | From total attempted messages how many consumers are not eligible to receive messages. |
+| 1 | Attempted | Total Outbound Messages/Total IVR Deflections sent to the Proactive Messaging / Connect To Messaging systems |
+| 2 | Eligible | From total attempted messages how many consumers are eligible to receive messages. |
```
[the remark this answers is not in the document]
Comment (jschatzman-lp) [reply pages/Documents/Reporting/OutboundReportingAPI/overview.md:42]: I added "For example, trying to send a SMS message to a landline will not count as eligible"

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md [hunk 3]
```diff
@@ -58,16 +57,13 @@ Outbound Reporting API provides the following data fields. This table explains t
...
 **What are the limitations**
 
-- Proactive campaign id data field is not currently available in transaction API responses as of now. It will be added soon.
-- Handoff Id, Handoff name data fields are not currently available in the Outbound Reporting API.
 - First message and override message data fields are not currently available in the Outbound Reporting API.
 - Total summary of eligibility, sent, delivered combined for all channels / skills per IVR outbound number is not currently available in the Outbound Reporting API.
-- Total summary of eligibility, sent, delivered combined for all channels / skills per account is not currently available in the Outbound Reporting API.
 - The capability of generating reports of all the consumers who previously opt out from Proactive Messaging to receive any future messages is not available in the Outbound Reporting API.
-- Maximum time duration for a transaction reporting api request cannot exceed 24 hours.
+- The maximum allowed time interval for a transaction reporting api request cannot exceed 24 hours.
```
[the remark this answers is not in the document]
Comment (jschatzman-lp) [reply pages/Documents/Reporting/OutboundReportingAPI/overview.md:63]: I don't think this is necessary.  We already mention this again the FAQ

```8. What date range is supported by the Outbound Reporting API to pull raw transactional data?

The caller can pull the data for a given 24 hour time interval from today or any previous day up to 13 months ago.

9. What date range is supported by the Outbound Reporting API to pull analytical data?

The caller can pull the data for a given 60 day time interval from today or any previous day up to 13 months ago.
```

Furthermore, brands will receive an API error that tells them if they used a time range that is too big.

**Comment by jschatzman-lp** [comment]
@itay1313 Yes, please merge it.  Thanks

**Review by itay1313 — APPROVED** [review]

**Pull request merged**